modules/rhel-adding-node.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 // * machine_management/adding-rhel-compute.adoc
+// * post_installation_configuration/node-tasks.adoc
```

Comment:
Not sure if this is the right place, some comments on the section: http://file.rdu.redhat.com/~ahardin/July102020/post-install-config/post_installation_configuration/node-tasks.html#post-install-config-adding-rhel-compute

1. In http://file.rdu.redhat.com/~ahardin/July102020/post-install-config/post_installation_configuration/node-tasks.html#rhel-preparing-playbook-machine_post-install-node-tasks  
```
Enable the repositories required by OpenShift Container Platform 4.5:

# subscription-manager repos \
    --enable="rhel-7-server-rpms" \
    --enable="rhel-7-server-extras-rpms" \
    --enable="rhel-7-server-ansible-2.8-rpms" \
    --enable="rhel-7-server-ose-4.5-rpms"
```
We're requiring ansible-2.9 instead of ansible-2.8 in OCP-4.4 and 4.5, 
https://bugzilla.redhat.com/show_bug.cgi?id=1828027

2. http://file.rdu.redhat.com/~ahardin/July102020/post-install-config/post_installation_configuration/node-tasks.html#installation-approve-csrs_post-install-node-tasks
We don't need to approve the csr for RHEL workers any more, this is completed in the scale-up playbook. So we don't need this during RHEL scale-up.

Reply:
https://github.com/openshift/openshift-docs/pull/24806 resolves the Ansible version issue